target/linux/bcm63xx/base-files/lib/upgrade/platform.sh

```diff
@@ -13,4 +23,50 @@ platform_check_image() {
 	esac
 }
 
-# use default for platform_do_upgrade()
+cfe_jffs2_upgrade_tar() {
+	local tar_file="$1"
+	local kernel_mtd="$(find_mtd_index $CI_KERNPART)"
+
+	if [ ! "$kernel_mtd" ]; then
```

Comment:
-z ?

Reply:
I will change it.
FYI, this was copied from https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/upgrade/nand.sh#L45

Reply:
Hmm, looks like this should be patched.